language-support/ts/codegen/tests/build-and-lint.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+YARN=$PWD/$1
+DAML2TS=$PWD/$2
+DAR=$PWD/$3
+TS_DIR=$PWD/$4
+
+TMP_DIR=$(mktemp -d)
+cleanup() {
+  rm -rf $TMP_DIR
+}
+trap cleanup EXIT
+
+cp -r $TS_DIR/* $TMP_DIR
+cd $TMP_DIR
+
+$DAML2TS -o generated/src $DAR
+$YARN install
+$YARN workspaces run build
+$YARN workspaces run lint
```

Comment:
Maybe this could be separated into several Bazel rules and a macro that ties them together. That way we could leave it to `rules_nodejs` to call `yarn`.
- One to call `daml2ts` - could be reused in other places that require generated ts.
- `ts_library`/`nodejs_binary` as appropriate to test the generated ts.
- Unfortunately, there doesn't seem to be linting support in `rules_nodejs`. So, that' need some custom `sh_test` still. See https://github.com/bazelbuild/rules_nodejs/issues/570.

Reply:
That's a good idea. I'll do that in a separate PR at some point in the future when I find time.